graphics/include/ignition/common/Material.hh

```diff
@@ -222,6 +242,14 @@ namespace ignition
       /// \return the lighting enabled state
       public: bool Lighting() const;
 
+      /// \brief Set the Physically Based Rendering (PBR) material
+      /// \return The PBR material to set to.
+      public: void SetPbrMaterial(const Pbr &_pbr);
+
+      /// \brief Get the Physically Based Rendering (PBR) material
+      /// \return Pointer to the PBR material. Null if it does not exist.
+      public: Pbr *PbrMaterial() const;
```

Comment:
I think this should return a const pointer, because the caller can use `SetPbrMaterial` to set it. This may be a good time to make the change and document on the migration guide.

Reply:
Agree. I'll follow up on this.